AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -513,14 +513,22 @@ def grid(property: str = 'name') -> str:
         """
         Grid component properties.
           - 'Grid Size': The size of the grid, default value is 32
-          - 'Secondary Grid Spacing': The spacing value for the secondary grid, i.e. 1.0
+          - 'Axis Color': Color of the grid axis, default value is 0,0,255 (blue)
```

Comment:
would be helpful to specify what type is expected here azlmbr.math.Color(r,g,b,a)
also for other color property paths.
azlmb might not be needed you could just say math.Color